### PR TITLE
COMMERCE-1887

### PR DESCRIPTION
**Pull request by corbinmurakami**

### CODE REVIEW

File: docs/commerce/2.x/en/developer-guide/tutorial/implementing-a-custom-low-stock-activity.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-# Implementing a Custom Low Stock Activity
+# Implementing a Custom Low Stock Action
```
Comment (corbinmurakami) [inline docs/commerce/2.x/en/developer-guide/tutorial/implementing-a-custom-low-stock-activity.md]: While the extension point is called Low Stock Activity, the actual feature is referred to as Low Stock Action in the UI. I didn't do an extensive audit, so an extensive audit is required to pick up all the instances of Low Stock Activity.

File: docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
```
Comment (corbinmurakami) [inline docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md]: There's a relationship with shipments and the order fulfillment lifecycle which is not mentioned at all here but should be mentioned.

File: docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Liferay Commerce provides the tools to manage product inventory for merchants with a single store, to organizations with multiple warehouses and stores. Store administrators can use these features to maintain quantities for sales and handle shipments to complete orders.
 
-There are several key features used in managing inventory: Warehouses, Availability Estimates, Low Stock Activity, and the ability to set inventory by warehouse.
+There are several key features used in managing inventory: Warehouses, Availability Estimates, Low Stock Actions, and the ability to set inventory by warehouse.
 
 ## Warehouses
```
Comment (corbinmurakami) [inline docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md]: A logical diagram of warehouses and channels would be helpful here.

File: docs/commerce/2.x/en/user-guide/catalog/warehouse-reference-guide.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ To manage your warehouses, navigate to the _Control Panel_ → _Commerce_ → _S
 | Latitude | Warehouse's Latitude |
 | Longitude | Warehouse's Longitude |
```
Comment (corbinmurakami) [inline docs/commerce/2.x/en/user-guide/catalog/warehouse-reference-guide.md]: There should be a mention of when and how a warehouse's geolocation is used and the perquisites for this to work.

File: docs/commerce/2.x/en/user-guide/catalog/availability-estimates.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Availability Estimates
 
-This article documents how to configure Availability Estimates. If a product becomes unavailable, store owners can provide customers with an estimate for when a product will be restocked. There are three default time periods in Liferay Commerce.
```
Comment (corbinmurakami) [inline docs/commerce/2.x/en/user-guide/catalog/availability-estimates.md]: Availability estimates are created by our initializers--they don't ship out of the box.

**Comment by justinchoi001** [comment]
Started reviewing.

File: docs/commerce/2.x/en/developer-guide/tutorial/implementing-a-custom-low-stock-activity.md (hunk 1 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/developer-guide/tutorial/implementing-a-custom-low-stock-activity.md]: Fixed. Should change the article title too.

Comment (justinchoi001) [reply docs/commerce/2.x/en/developer-guide/tutorial/implementing-a-custom-low-stock-activity.md]: Furthermore, we have multiple articles that refer to this; I am going to change them them all.

File: docs/commerce/2.x/en/user-guide/catalog/availability-estimates.md (hunk 5 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/user-guide/catalog/availability-estimates.md]: Fixed.

File: docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md (hunk 3 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md]: @corbinmurakami , I've created a trello card for this. We're going to have to come back to this when we have more time. In the meantime, do you have an existing image?

File: docs/commerce/2.x/en/user-guide/catalog/availability-estimates.md (hunk 5 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/user-guide/catalog/availability-estimates.md]: Fixed.

File: docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md (hunk 2 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md]: Fixed.

File: docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md (hunk 3 above)
Comment (corbinmurakami) [reply docs/commerce/2.x/en/user-guide/catalog/introduction-to-managing-inventory.md]: No, I don't. Sorry, and thanks for adding it to the backlog.

File: docs/commerce/2.x/en/user-guide/catalog/warehouse-reference-guide.md (hunk 4 above)
Comment (justinchoi001) [reply docs/commerce/2.x/en/user-guide/catalog/warehouse-reference-guide.md]: Fixed.

**Comment by jrhoun** [comment]
Closing this since @justinchoi001 sent me https://github.com/jrhoun/liferay-learn/pull/43

**Pull request closed without merging**